packages/core/src/modules/vc/W3cCredentialService.ts

```diff
@@ -35,17 +34,17 @@ import { W3cCredentialRecord, W3cCredentialRepository } from './repository'
 @injectable()
 export class W3cCredentialService {
   private w3cCredentialRepository: W3cCredentialRepository
-  private didResolver: DidResolverService
   private suiteRegistry: SignatureSuiteRegistry
+  private w3cVcModuleConfig: W3cVcModuleConfig
 
   public constructor(
     w3cCredentialRepository: W3cCredentialRepository,
-    didResolver: DidResolverService,
-    suiteRegistry: SignatureSuiteRegistry
+    suiteRegistry: SignatureSuiteRegistry,
```

Comment:
```suggestion
    signatureSuiteRegistry: SignatureSuiteRegistry,
```

I think it would be good to be a bit more explicit here.